libs/langchain/langchain/document_loaders/audio.py

```diff
@@ -0,0 +1,24 @@
+from __future__ import annotations
+
+from pathlib import Path
+from typing import Any, Union
+
+from langchain.document_loaders.blob_loaders import FileSystemBlobLoader
+from langchain.document_loaders.generic import GenericLoader
+from langchain.document_loaders.parsers.audio import AzureSpeechServiceParser
+
+
+class AzureSpeechServiceLoader(GenericLoader):
```

Comment:
Made a PR here to help with the pattern for Generic Loader: https://github.com/langchain-ai/langchain/pull/14004 -- we don't want any file logic to live outside of file system blob loader.

@kzmain met know if you want to complete the refactor or if you want me to do it

Reply:
@kzmain 

1) I updated the service loader to use generic loader (see https://github.com/langchain-ai/langchain/pull/14004)
2) I expanded the kwargs so the parameters are easy to discover and refactored some of the resolving of values

Would you be able to test that the service still works for you? I don't have credentials set up so testing help will be helpful


3) Could you  run the parser with 2 audio files at once using `from_filesystem` to confirm that transcribing more than one file at a time works?